### PR TITLE
some review on part 1

### PR DESCRIPTION
**Pull request by bam241**

### CODE REVIEW

File: PAPER/FIT_Paper_Ex1/Part_1.tex [hunk 1]
```diff
@@ -1,52 +1,56 @@
...
 
-The different institutions that use fuel cycle simulations pursue different
-goals. Consequently, their computational tools vary in term of complexity. The
-level of complexity of each software has to be coherent in regards to the
-different studies it is used for. To improve confidence in the results,
-institutions are tempted to increase the complexity of their software even if
-this complexity might not be necessary.
+The different institutions that use (and develop) fuel cycle simulators pursue
+different goals. Consequently, their computational needs vary in term of
+complexity and the level of complexity of each software pieces has to be in
```
Comment (thiollie) [inline PAPER/FIT_Paper_Ex1/Part_1.tex]: Problem with grammar srtucture...?

File: PAPER/FIT_Paper_Ex1/Part_1.tex [hunk 2]
```diff
@@ -1,52 +1,56 @@
...
+adequation with regards to their different simulations requirements. To improve
```
Comment (thiollie) [inline PAPER/FIT_Paper_Ex1/Part_1.tex]: Replace "simulation requirement" with a term that say "what do you want to do with your soft. What is the study you want to realize." This was said before with "different studies it is used for". Maybe we should find better.

File: PAPER/FIT_Paper_Ex1/Part_1.tex [hunk 3]
```diff
@@ -1,52 +1,56 @@
...
-question associated to a targeted precision, some functionality are needed while
-some others are not. Knowing the importance of each ones helps to choose an
-appropriate software or to engage code developments to solve the question. Also,
-some technical issues are assessed by numerous studies performed with different
-software and it is often difficult to compare them. Knowing the impact of
-functionality on different simulation outputs helps to estimate the confidence
-level of studies.
+technical question the code assesses: solving a given technical question
+associated with a targeted precision will required a limited set of
+functionality complexity. Knowing the importance of each ones helps to choose an
```
Comment (thiollie) [inline PAPER/FIT_Paper_Ex1/Part_1.tex]: Don't understand the "limited". If possible keep the idea of "some functionality are needed while
some others are not"...

File: PAPER/FIT_Paper_Ex1/Part_1.tex [hunk 4]
```diff
@@ -1,52 +1,56 @@
...
+appropriate software or to engage code developments to solve a specific
+question. Also, some technical issues are assessed by numerous studies performed
+with different software and it is often difficult to compare them. Knowing the
+impact of functionality on different simulation outputs helps to rank them by
```
Comment (thiollie) [inline PAPER/FIT_Paper_Ex1/Part_1.tex]: replace "rank" by a good way to write "to estimate the confidence level of studies.".

File: PAPER/FIT_Paper_Ex1/Part_1.tex [hunk 5]
```diff
@@ -1,52 +1,56 @@
...
-The FIT (Functionality Isolation Test) Project has been conceived for this
-purpose. The project goal is to determine the minimum level of details in fuel
-cycle simulations that has to be taken into account according to the type of
-study and the wanted confidence level. Among different functionality of
-interest, this first work focuses on the ability of fuel cycle software to adapt
-the fresh fuel composition regarding to the available material for fuel
-fabrication.
+The FIT (Functionality Isolation Test) Project has been conceived to understand
+the circumstances under which the choice of algorithm and/or model influences
+the conclusions that one might draw from such a fuel cycle simulation. Ths
```
Comment (thiollie) [inline PAPER/FIT_Paper_Ex1/Part_1.tex]: Ths -> The

File: PAPER/FIT_Paper_Ex1/Part_1.tex [hunk 6]
```diff
@@ -1,52 +1,57 @@
...
-cycle tools can be used to study current nuclear fleet for industrial operation
-optimization. Those tools can also be used to assess the future of nuclear
-energy and provide then informations for political decision process. Finally,
-those tools may be used for Research and Development training as they give an
-access to key point data related to the fuel cycle. 
+Fuel cycle simulator are used worldwide for wide range of applications:
+optimisation of the industrial operation of a existing nuclear fleet, assessing
+the future of nuclear energy and providing valuable informations required for
+the political decision, evaluation/verification of the operation of an exisiting
+nuclear fleet by national or international safety authoritesy. Moreover, those
```
Comment (doligez) [inline PAPER/FIT_Paper_Ex1/Part_1.tex]:
```suggestion
nuclear fleet by national or international safety authorities. Moreover, those
```

File: PAPER/FIT_Paper_Ex1/Part_1.tex [hunk 7]
```diff
@@ -1,52 +1,57 @@
...
-cooling pool and waste disposal. Those tools help to identify drivers and
-interactions between parameters in fuel cycle physic. For that, they often
-implement physic models for different key points of the cycle like fuel
-fabrication or burn up depletion calculations. 
+A fuel cycle simulator aims to models an entire nuclear fleet including main
+facilities, such as nuclear reactors, fuel fabrication plants, reprocessing
+plants, cooling pool and/or waste disposal. Those tools help to identify drivers
+and interactions between parameters in fuel cycle. They implement physic models
+for different key points of the cycle such as fuel fabrication or burn up
+depletions, with various level of complexicity.
```
Comment (doligez) [inline PAPER/FIT_Paper_Ex1/Part_1.tex]: A fuel cycle simulator aims to models an entire nuclear fleet including main
facilities, such as nuclear reactors, fuel fabrication plants, reprocessing
plants, cooling pool and/or waste disposal. They implement physic models
for different key points of the cycle such as fuel fabrication or burn up
depletions, with various level of complexicity. Those tools help to identify drivers
and interactions between parameters in fuel cycle.

File: PAPER/FIT_Paper_Ex1/Part_1.tex [hunk 8]
```diff
@@ -1,52 +1,57 @@
...
+tools are used for Research and Development training as access point to key
+datas related to the fuel cycle. 
 
-The different institutions that use fuel cycle simulations pursue different
-goals. Consequently, their computational tools vary in term of complexity. The
-level of complexity of each software has to be coherent in regards to the
-different studies it is used for. To improve confidence in the results,
-institutions are tempted to increase the complexity of their software even if
-this complexity might not be necessary.
+The different institutions that use (and develop) fuel cycle simulators pursue
```
Comment (doligez) [inline PAPER/FIT_Paper_Ex1/Part_1.tex]:
```suggestion
The different institutions that use fuel cycle simulators pursue
```

File: PAPER/FIT_Paper_Ex1/Part_1.tex [hunk 9]
```diff
@@ -1,52 +1,57 @@
...
+different goals. Consequently, their computational needs vary in term of
+complexity and the level of complexity of each software pieces has to be in
```
Comment (doligez) [inline PAPER/FIT_Paper_Ex1/Part_1.tex]:
```suggestion
complexity and the level of precision of each software pieces has to be in
```

File: PAPER/FIT_Paper_Ex1/Part_1.tex [hunk 10]
```diff
@@ -1,52 +1,57 @@
...
+adequation with regards to their different simulations requirements. To improve
+confidence in the results, institutions are tempted to increase the complexity
+of their software even if this complexity might not be necessary.
```
Comment (doligez) [inline PAPER/FIT_Paper_Ex1/Part_1.tex]: bcp de fois le mot "complexity"

File: PAPER/FIT_Paper_Ex1/Part_1.tex (hunk 3 above)
Comment (doligez) [reply PAPER/FIT_Paper_Ex1/Part_1.tex]: I agree with Nicolas

File: PAPER/FIT_Paper_Ex1/Part_1.tex (hunk 4 above)
Comment (doligez) [reply PAPER/FIT_Paper_Ex1/Part_1.tex]: OK pour la proposition de Nico

File: PAPER/FIT_Paper_Ex1/Part_1.tex (hunk 2 above)
Comment (bam241) [reply PAPER/FIT_Paper_Ex1/Part_1.tex]: simulations needs ?

Comment (bam241) [reply PAPER/FIT_Paper_Ex1/Part_1.tex]: simulation's goals ?

**Pull request merged**